src/media/class-wpml-beaver-builder-media-hooks.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+class WPML_Beaver_Builder_Media_Hooks implements IWPML_Action {
```

Comment:
@strategio You will end up with exactly the same class in Elementor with the only difference being the `KEY` and the `create_media_updater` function. You should create an abstract base class in PB for this.

Reply:
@strategio I can't see how this has been addressed in the other pull request.

Reply:
@brucepearson, I wrote a comment on this last week but it's seems it was lost (this is not the first time it happens to me).

What I am trying to do is avoid inheritance between packages. IMHO, this is not a good strategy because we are testing in one package the code which was written in another one. So far, I tried to use composition instead when we have shared code. For now, with this small class, I don't see a real need to use do this. But we can reevaluate later if we have more hooks following the same pattern for multiple page builders.

Reply:
@strategio The other class in this extends a class from page builders - `WPML_Beaver_Builder_Update_Media extends WPML_Page_Builders_Update_Media`
I don't see this class being any different.

Reply:
I changed it in the following PR https://github.com/OnTheGoSystems/wpml-page-builders/pull/23/files#diff-c6b5d5c2c50413da4633cc843e7faa55.